src/Microsoft.DotNet.Darc/DarcLib/Local.cs

```diff
@@ -97,10 +98,10 @@ public async Task UpdateDependenciesAsync(List<DependencyDetail> dependencies, I
             try
             {
                 IRemote arcadeRemote = await remoteFactory.CreateRemoteAsync(arcadeItem.RepoUri);
-                List<GitFile> engCommonFiles = await arcadeRemote.GetCommonScriptFilesAsync(arcadeItem.RepoUri, arcadeItem.Commit, repoIsVmr);
+                List<GitFile> engCommonFiles = await arcadeRemote.GetCommonScriptFilesAsync(arcadeItem.RepoUri, arcadeItem.Commit, mapping);
                 // If we're updating arcade from a VMR build, the eng/common files will be in the src/arcade repo
                 // so we need to strip the src/arcade prefix from the file paths.
-                if (repoIsVmr)
+                if (mapping == VmrInfo.ArcadeMappingName)
```

Comment:
Is this correct? Can we make it more obvious how we use the `mapping` variable here? Seems like we are abusing `string` as a `boolean` and it's quite hard to parse. We only set it to `arcade` or `null`, probably because we are sending it to `GetCommonScriptFilesAsync` but I think that's not good. We should keep a boolean like before and then only send `repoIsVmr ? VmrInfo.ArcadeMappingName : null` to methods?